src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs

```diff
@@ -1,85 +1,109 @@
         }
 
-      //  [Event(14, Level = EventLevel.Informational, Message = "Id {0}")]
         public void ConnectionStop(long connectionId)
         {
-       //     WriteEvent(14, connectionId);
+            _logger.LogDebug(14, $"{nameof(ConnectionStop)} -> Id: {connectionId}");
         }
 
-
-   //     [Event(4, Message = "Id {0} Status {1}")]
         internal void ConnectionRead(long connectionId, int status)
         {
-     //       WriteEvent(4, connectionId, status);
+            _logger.LogDebug(4, $"{nameof(ConnectionRead)} -> Id: {connectionId}, Status: {status}");
         }
 
- //       [Event(5, Message = "Id {0}")]
         internal void ConnectionPause(long connectionId)
         {
-   //         WriteEvent(5, connectionId);
+            _logger.LogDebug(5, $"{nameof(ConnectionPause)} -> Id: {connectionId}");
         }
 
- //       [Event(6, Message = "Id {0}")]
         internal void ConnectionResume(long connectionId)
         {
-   //         WriteEvent(6, connectionId);
+            _logger.LogDebug(6, $"{nameof(ConnectionResume)} -> Id: {connectionId}");
         }
 
-  //      [Event(7, Message = "Id {0}")]
         internal void ConnectionReadFin(long connectionId)
         {
-    //        WriteEvent(7, connectionId);
+            _logger.LogDebug(7, $"{nameof(ConnectionReadFin)} -> Id: {connectionId}");
         }
 
-//        [Event(8, Message = "Id {0} Step {1}")]
         internal void ConnectionWriteFin(long connectionId, int step)
         {
-  //          WriteEvent(8, connectionId, step);
+            _logger.LogDebug(8, $"{nameof(ConnectionWriteFin)} -> Id: {connectionId}, Step: {step}");
         }
 
- //       [Event(9, Message = "Id {0}")]
         internal void ConnectionKeepAlive(long connectionId)
         {
-   //         WriteEvent(9, connectionId);
+            _logger.LogDebug(9, $"{nameof(ConnectionKeepAlive)} -> Id: {connectionId}");
         }
 
- //       [Event(10, Message = "Id {0}")]
         internal void ConnectionDisconnect(long connectionId)
         {
-   //         WriteEvent(10, connectionId);
+            _logger.LogDebug(10, $"{nameof(ConnectionDisconnect)} -> Id: {connectionId}");
         }
 
-  //      [Event(11, Message = "Id {0} Count {1}")]
         internal void ConnectionWrite(long connectionId, int count)
         {
-    //        WriteEvent(11, connectionId, count);
+            _logger.LogDebug(11, $"{nameof(ConnectionWrite)} -> Id: {connectionId}, Count: {count}");
         }
 
- //       [Event(12, Message = "Id {0} Status {1}")]
         internal void ConnectionWriteCallback(long connectionId, int status)
         {
-   //         WriteEvent(12, connectionId, status);
+            _logger.LogDebug(12, $"{nameof(ConnectionWriteCallback)} -> Id: {connectionId}, Status: {status}");
+        }
+
+        void ILogger.Log(LogLevel logLevel, int eventId, object state, Exception exception, Func<object, Exception, string> formatter)
```

Comment:
Might as well make this public for consistency once we get rid of the static property.